src/Pages/Overview/Overview.js

```diff
@@ -1,7 +1,150 @@
 import React, { Component } from 'react'
+import './Overview.css'
+import axios from 'axios'
+import OverviewProfile from './OverviewProfile.js'
+
+export default class OverviewBoard extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      // All users array, update by callDatabase
+      users: [],
+      search: ''
+    }
+  }
+
+  componentDidMount () {
+    const token = window.localStorage
+      ? window.localStorage.getItem('jwtToken')
+      : ''
+
+    // Immediately direct to /login if no jwtToken token present
+    if (!token) {
+      if (this.props.history) this.props.history.push('/login')
+      return
+    }
+
+    // verify that user is login
+    axios
+      .post('/api/user/verify', { token })
+      .then(res => {
+        this.setState(
+          {
+            currentUser: res.data.email,
+            currentUserLevel: res.data.accessLevel,
+            authToken: token
+          },
+          () => {
+            this.callDatabase()
+          }
+        )
+      })
+      .catch(() => {
+        if (this.props.history) this.props.history.push('/login')
+      })
+  }
+
+  callDatabase () {
+    axios
+      // get all user!
+      .post('/api/user/users', {
+        // don't need email
+        token: this.state.authToken
+      })
+      .then(result => {
+        if (result.status >= 200 && result.status < 300) {
+          this.setState({ users: result.data })
+        }
+      })
+      .catch(err => {
+        console.log(err)
+      })
+  }
+
+  updateSearch (event) {
```

Comment:
what if i want to search by last name, email, membership type? If you don't want to add that now, make an issue for adding those search terms

Reply:
I will create an issue for that, this could be one of the update